SConstruct

```diff
@@ -13,6 +13,13 @@ from lsst.ci.hsc.gen2.validate import (RawValidation, DetrendValidation, SfmVali
                                        WriteObjectValidation, TransformObjectValidation,
                                        ConsolidateObjectValidation)
 
+try:
```

Comment:
Please add an inline comment so that when the assignee of the future DM-22256 can easily find this section to remove it: e.g.
`# TODO : workaround for DM-22256`
and again at the `if not havePyArrow:` below.